### ISSUE TITLE
Functions in expression data no longer called

### ISSUE THREAD

**Issue by matthewp** [bitovi/canjs#1839]
I have an element like:

```handlebars
<input name="phone" type="text" can-keyup="{setPhoneValue @element.val}">
```

setPhoneValue being a viewModel function. Previously `@element.val` would be called as a function but now instead the function itself is passed into setPhoneValue.  I assume this is related to the new subexpression behaviors.

**Comment by justinbmeyer** [comment]
I'll take a look today.

**Comment by justinbmeyer** [comment]
btw, I think:

```
{setPhoneValue (@element.val)}
```

might work.  This is sorta the reason I think the other syntax would be better:

```
{setPhoneValue @element.val()}
```

Because I think in 3.0, the only way to have a function run automatically is if you call it.  This means there will be lots of `(prettyDate)` and not `prettyDate()`.

**Comment by justinbmeyer** [comment]
@matthewp that pull request fixes this if you need it.

**Comment by daffl** [comment]
Closed in minor via #1840
